### PR TITLE
[appveyor] Upload ninja.exe as artifacts

### PR DESCRIPTION
**Pull request by atetubou**
I followed this document.
https://www.appveyor.com/docs/packaging-artifacts/#pushing-artifacts-from-scripts

This is for #1509

### CODE REVIEW

**Comment by atetubou** [comment]
artifacts is uploaded in
https://ci.appveyor.com/project/nico/ninja/build/job/3b6inii5iisr6kes/artifacts

**Comment by jhasse** [comment]
Thanks! Will this also attach the binary to a tag / release on GitHub?

**Comment by atetubou** [comment]
OK. I will try this.
https://www.appveyor.com/docs/deployment/github/

**Comment by atetubou** [comment]
I updated config, artifact will be released like
https://github.com/atetubou/ninja/releases/tag/1.0.39

But not sure this will work with travis.

File: appveyor.yml [hunk 1]
```diff
@@ -37,4 +37,15 @@ for:
 
         python misc/ninja_syntax_test.py
 
+        appveyor PushArtifact ninja.exe
+    deploy:
+      description: release
+      provider: GitHub
+      auth_token:
+        secure: wyu/fhbEU2AiTPWJNIkimoVCBquTpTeVHOUBGQbB3m0GRS2ch+plZ7gtQNgvanJ0
```
Comment (atetubou) [inline appveyor.yml]: Please change here following
https://www.appveyor.com/docs/deployment/github/#provider-settings

**Comment by jhasse** [comment]
As the AppVeyor project belongs to nico, it looks to me that he's the only one who can encrypt a personal access token.

@nico Can you do that? Another solution would be to create a new AppVeyor account with the ninja-build organisation.

**Comment by jhasse** [comment]
We have now switched to GitHub Actions which makes things a bit easier.

**Pull request closed without merging**